load-testing/initial-load-testing.md

```diff
@@ -0,0 +1,89 @@
+# Initial load testing
+
+## Files used
+
+A 3Gb, 40000 second wav file, created using ffmpeg.
+
+2 2.8Gb bmp files created using ImageMagick's convert tool
+
+2000 PDFs created using ImageMagick's convert tool
+
+10000 PDFs created using ImageMagick's convert tool
+
+These files have been uploaded to the tdr-upload-test-data bucket in the Sandbox environment if anyone wants to use them.
+
+## Running the tests
+
+I carried out the upload on a t3.medium EC2 instance with 2 CPU cores and 4GB RAM. Our first users will have a similar CPU with 8GB of RAM so the test machine is lower spec.
```

Comment:
Which browser(s) did you use?

Reply:
I've mentioned in note 6 that I used Chrome and Firefox when they failed but I've put a line in here to say it so it's clearer up front.